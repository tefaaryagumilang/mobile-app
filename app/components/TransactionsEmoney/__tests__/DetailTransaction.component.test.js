import React from 'react';
import renderer from 'react-test-renderer';
import TransactionItem from '../DetailTransactionEmoney.component';

describe('DetailTransaction component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TransactionItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
