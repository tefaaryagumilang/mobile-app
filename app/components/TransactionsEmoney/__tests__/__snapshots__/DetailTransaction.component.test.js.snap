// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailTransaction component renders correctly 1`] = `
<View
  keyboardShouldPersistTaps="handled"
>
  <View
    style={
      Object {
        "backgroundColor": "#ccc",
        "height": 1,
      }
    }
  />
  <View
    style={
      Array [
        Object {
          "paddingHorizontal": 20,
        },
        Object {
          "paddingHorizontal": 30,
          "paddingTop": 15,
        },
        Object {
          "backgroundColor": "#ffffff",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "paddingHorizontal": 20,
        },
        Object {
          "paddingHorizontal": 30,
          "paddingTop": 15,
        },
        Object {
          "backgroundColor": "#ffffff",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#ccc",
          "height": 1,
        }
      }
    />
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            ID Transaksi
          </Text>
        </View>
        <View>
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "700",
                },
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            Waktu
          </Text>
        </View>
        <View>
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "700",
                },
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            Invalid date
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            Jumlah
          </Text>
        </View>
        <View>
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "700",
                },
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
             
            --
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            Dari
          </Text>
        </View>
        <View>
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "700",
                },
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            Ke
          </Text>
        </View>
        <View>
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "700",
                },
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingVertical": 10,
        }
      }
    >
      <View>
        <View>
          <Text
            style={
              Array [
                undefined,
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            Keterangan
          </Text>
        </View>
        <View>
          <Text
            style={
              Array [
                Object {
                  "fontWeight": "700",
                },
                Object {
                  "color": "#000",
                  "fontSize": 14,
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
